data_management/opensearch/tests/test_process_and_index_file.py

```diff
@@ -0,0 +1,197 @@
+import tempfile
+from unittest import mock
+from uuid import uuid4
+
+import pytest
+from opensearchpy import RequestsHttpConnection
+from sqlalchemy import (
+    Boolean,
+    Column,
+    DateTime,
+    ForeignKey,
+    String,
+    Text,
+    create_engine,
+)
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import declarative_base, relationship, sessionmaker
+
+from data_management.opensearch.index_file_content_and_metadata_in_opensearch import (
+    index_file_content_and_metadata_in_opensearch,
+)
+
+Base = declarative_base()
+
+
+class Body(Base):
```

Comment:
I'd love to avoid redefining this here as we have apppp/models.py but we use flask-sqlalchemy in there. We will need to do some work to make models.py look like this , i.e not flask specific but that might cause some issues in our queries in the webapp. Can create a techdebt ticket for the backlog.

Reply:
https://national-archives.atlassian.net/browse/AYR-1245?atlOrigin=eyJpIjoiM2RhOWFkODczZTU2NGFlOGE2ZTc3MmVlYTBmYTNlNzUiLCJwIjoiamlyYS1zbGFjay1pbnQifQ